conf/modules.config

```diff
@@ -637,6 +637,41 @@ process {
         ]
     }
 
+    withName: GANON_CLASSIFY {
+        ext.args = params.ganon_save_readclassifications ? { "${meta.db_params} --output-all --output-lca --output-unclassified" } : { "${meta.db_params}" }
```

Comment:
Yes - to get all the per-read classification you need to provide all of those (just with slightly different information), we can stick them it all in one that's fine with `--output-single`! I include these here as they influence output files (rather than the execution of the tool itself).

Reply:
Wait update:

Yes - to get all the per-read classification you need to provide all of those (just with slightly different information).

I didn't include `--output-single` as that seemed to be related to when you [supply multiple databases at once](https://pirovc.github.io/ganon/classification/#multiple-and-hierarchical-classification), something we don't support currently. Regardless a user can still supply that with `db_params` if they really want.